### PR TITLE
Fixes Issue-310 :Added contributing guidelines related to hijacking owned i…

### LINKED ISSUE #310: Adding a couple of ground rules to contribution.md

### PR DESCRIPTION
**Pull request by abhiii5459**
…ssues.

Fixes #310 .
Nobody should hijack another person's claimed issue unless deemed as necessary to be  marked unclaimed .
If within 2 days, no action is seen to be happening on a claimed issue, it will be marked "not-claimed".

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -7,21 +7,14 @@
...
 2. Say hello on the [#codebuddies-meta channel in the Slack](https://codebuddiesmeet.slack.com/messages/codebuddies-meta/). Feel free to ask questions here, or to ask for someone to review your pull request.
 3. [Install Meteor](https://www.meteor.com/install) and Node: [docs.npmjs.com/cli/install](https://docs.npmjs.com/cli/install) if you don't already have them installed.
-4. Please star this repository! We need to reach 100 stars so that we can apply to the [Open Collective](https://opencollective.com/opensource/apply).
+4. Please star this repository! We need to reach 100 stars so that we can apply to the [Open Collective](https://opencollective.com/opensource/apply). [Edit - We're already there!]
 5. Fork this repository! Once you have a copy of this repo on your own account, clone this repo to your computer by typing in something like:
   `git clone https://github.com/codebuddiesdotorg/cb-v2-scratch.git`
 (Replace the URL with your own repository URL path.)
-6. Setup this repository as an upstream branch using:
```
Comment (lpatmo) [inline contributing.md]: Looks generally great, @abhiii5459, except this bit overrides an earlier merge of a PR that edited Step #6, with the instructions about how to set up your forked branch to have an upstream pointing back to the original CB repo. :)

Can you check if you've got it set up? Then you should be able to `git fetch upstream` `git checkout master` `git merge upstream/master` to merge in the latest changes from master.

Thanks!

Comment (abhiii5459) [inline contributing.md]: Will do!

**Comment by abhiii5459** [comment]
@lpatmo  Done!

**Comment by lpatmo** [comment]
Thank you @abhiii5459!

**Pull request merged**